claim/claim_test.go

```diff
@@ -209,3 +210,24 @@ func TestClaimSchema(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func TestULID(t *testing.T) {
+	// Validate that the ULID function is thread-safe and generates
+	// monotonically increasing values
+
+	var wg sync.WaitGroup
+	for i := 0; i < 100; i++ {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			var last string
+			for j := 0; j < 1000; j++ {
+				next := ULID()
+				if strings.Compare(next, last) != 1 {
+					t.Fatal("generated a ULID that was not monotonically increasing")
+				}
```

Comment:
We need to set `last` to `next` here or somewhere before we exit this for loop, right?  As I'm reading this, currently it seems `last` will always be an empty string.

Reply:
bwahahahhahaha 🤦 Thanks Vaughn!